drivers/jwt/Kconfig

```diff
@@ -0,0 +1,27 @@
+menuconfig JWT
```

Comment:
Why this is in driver subdirectory, could we make it a library as you are calling it a library in commit msg?

Reply:
i actually did mean this to be moved to library.  I'll fix that.